content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,186 @@
+  line` to add *users* to this group.
+- :guilabel:`Inherited` tab: inherited means that users added to this group are automatically added
+  to the groups listed on this tab. Click :guilabel:`Add a line` to add *inherited groups*.
+
+  .. example::
+     For example, if the group *Sales/Administrator* lists the group *Website/Restricted Editor* in
+     its :guilabel:`Inherited` tab, then any users added to the *Sales/Administrator* group
+     automatically receive access to the *Website/Restricted Editor* group, as well.
+
+- :guilabel:`Menus` tab: defines which menus/models the group can have access to. Click
+  :guilabel:`Add a line` to add a specific *menu*.
+- :guilabel:`Views` tab: lists which views in Odoo the group has access to. Click :guilabel:`Add a
+  line` to add a *view* to the group.
+- :guilabel:`Access Rights` tab: lists the first level of rights (models) that this group has access
+  rights to. Click :guilabel:`Add a line` to link *access rights* to this group. In this tab, the
+  :guilabel:`Model` column represents the common name of the menu/model, and the :guilabel:`Name`
+  column represents the technical name given to the model. For each model, enable the following
+  options as appropriate:
+
+  - :guilabel:`Read`: users can see the object's existing values.
+  - :guilabel:`Write`: users can edit the object's existing values.
+  - :guilabel:`Create`: users can create new values for the object.
+  - :guilabel:`Delete`: users can delete values for the object.
+
+  .. tip::
+     First try searching for the common name of the model in the drop-down menu of the
+     :guilabel:`Model` column. The :guilabel:`Model` technical name can be found by expanding the
+     model common name by clicking the :guilabel:`expand` button, which is represented by a
+     :guilabel:`external link` icon.
+
+     The model technical name can also be accessed in :ref:`developer mode <developer-mode>`.
+
+     On a form, navigate to any field, and hover over the field name. A box of backend information
+     reveals itself with the specific Odoo :guilabel:`Object` name in the backend. This is the
+     technical name of the model that should be added.
+
+     .. image:: access_rights/technical-info.png
+        :align: center
+        :alt: Technical information shown on a field of a model, with object highlighted.
+
+- :guilabel:`Record Rules`: lists the second layer of editing and visibility rights.
+  :guilabel:`Record Rules` overwrite, or refine, the group's *Access Rights*. Click :guilabel:`Add a
+  line` to add a *record rule* to this group. For each rule, choose values for the following
+  options:
+
+  - :guilabel:`Apply for Read`.
+  - :guilabel:`Apply for Write`.
+  - :guilabel:`Apply for Create`.
+  - :guilabel:`Apply for Delete`.
+
+  .. important::
+     Record rules are written using a *Domain*, or conditions that filter data. A domain expression
```

Comment:
Should this first use of Record rules be applied with a GUI, as well? The other uses of that term in this admonition are using a GUI. What do you think?